design_docs/Helpers.md

```diff
@@ -0,0 +1,44 @@
+# Helpers
+
+The [helpers](../helpers) directory is designed to be a collection place for single purpose scripts, useful
+in setting up or configuring the analysis, but not a direct part of the end-to-end workflow.
+
+## Pedigree
+
+Running accurate genetic analysis is heavily dependent on an accurate representation of relationships between samples.
+For the CPG, this data is typically stored within the sample-metadata database, accessible through the corresponding API
+client.
+
+The [Pedigree-generation](../helpers/pedigree_from_sample_metadata.py) script has been included which
+will query the sample-metadata API for a given cohort, and rearrange the results into two outputs:
+
+- A Pedigree
+- A mapping of the external individual IDs to CPG sample IDs
+
+Additional options can be supplied to:
+
+- export the Pedigree as PLINK format instead of .ped format
+- export the Pedigree with all individuals as singletons
+
+### Pedigree manipulation
+
+The Pedigree data pulled directly from the SM-API contains all members and relationships in the context
+of the participants' external identifiers (PID). This can't be matched up with the VCF contents, as in
+sequence data files processing is instead done in the context of the Sample IDs (SID).
+
+To complicate things, there is no guarantee of a 1:1 relationship between participants and samples (e.g.
+replacement samples, but also Genome and Exome library preparations would be considered separate samples).
```

Comment:
This assumption isn't true, most of the time genomes and exomes will have the same sample ID, but different sequence IDs.